view/theme/frio/templates/paginate.tpl

```diff
@@ -0,0 +1,13 @@
+{{if $pager}}

```

Comment:
a short comment in the format `{{* description *}} with a short template description for newbies would be good